api/src/test/java/com/epam/pipeline/test/creator/docker/DockerCreatorUtils.java

```diff
@@ -158,6 +159,18 @@ public static DockerRegistry getDockerRegistry(final Long id, final String owner
         return dockerRegistry;
     }
 
+    public static DockerRegistry getDockerRegistry(final Long id, final String path,
```

Comment:
Could we use methods overloading here? (and for `tool` and `toolGroup` creators)

Reply:
No, there are no suitable methods in creators to reduce the code.

It will only be increased, because I would have to remove irrelevant object properties by setting null to them after calling already existing methods.

That's an necessity for my Mockito.eq() to work. As it compares every property in object.

Reply:
As I can see
```
    public static DockerRegistry getDockerRegistry(final Long id, final String owner) {
        return getDockerRegistry(id, TEST_STRING, owner, TEST_STRING);
    }

    public static DockerRegistry getDockerRegistry(final Long id, final String path,
                                                   final String owner, final String externalUrl) {
        final DockerRegistry registry = new DockerRegistry();
        registry.setId(id);
        registry.setPath(path);
        registry.setOwner(owner);
        registry.setExternalUrl(externalUrl);
        registry.setCaCert(TEST_STRING);
        registry.setDescription(TEST_STRING);
        registry.setHasMetadata(true);
        registry.setPassword(TEST_STRING);
        registry.setSecretName(TEST_STRING);
        registry.setUserName(TEST_STRING);
        return registry;
    }
```
works

Reply:
Yeah, you are right. I thought that it would affect my code, but it doesn't. I suppose, that I was thinking about some another test, writing the comment above, where I definitely will have such a problem :) So I fixed this issue, as you recommended at the first time.